Remotion/Web/Development.WebTesting/Utilities/AssertionExceptionUtility.cs

```diff
@@ -28,26 +28,44 @@ public static class AssertionExceptionUtility
   {
     [NotNull]
     [MustUseReturnValue]
-    public static Exception CreateControlDisabledException ()
+    public static WebTestException CreateControlDisabledException ()
     {
-      return new MissingHtmlException ("The control is currently in a disabled state. Therefore, the operation is not possible.");
+      return new WebTestException ("The control is currently in a disabled state. Therefore, the operation is not possible.");
     }
 
     [NotNull]
     [MustUseReturnValue]
-    public static Exception CreateControlDisabledException ([NotNull] string controlName)
+    public static WebTestException CreateControlDisabledException ([NotNull] string controlName)
     {
       ArgumentUtility.CheckNotNullOrEmpty ("controlName", controlName);
 
-      return new MissingHtmlException (
+      return new WebTestException (
           string.Format ("The '{0}' is currently in a disabled state. Therefore, the operation is not possible.", controlName));
     }
 
     [NotNull]
     [MustUseReturnValue]
-    public static Exception CreateControlReadOnlyException ()
+    public static WebTestException CreateControlReadOnlyException ()
     {
-      return new MissingHtmlException ("The control is currently in a read-only state. Therefore, the operation is not possible.");
+      return new WebTestException ("The control is currently in a read-only state. Therefore, the operation is not possible.");
+    }
+
+    [NotNull]
+    [MustUseReturnValue]
+    public static WebTestException CreateControlMissingException ([NotNull] MissingHtmlException missingHtmlException)
+    {
+      ArgumentUtility.CheckNotNull ("missingHtmlException", missingHtmlException);
+
+      return new WebTestException ("The element cannot be found.", missingHtmlException);

```

Comment:
It would be possible to pass the `Coypu.MissingHtmlException` message, containing the selector (`Unable to find css: <selector>`, `Coypu.AmbiguousException` is similar). This would then result in:

```csharp
[NotNull]
[MustUseReturnValue]
public static WebTestException CreateControlMissingException ([NotNull] string exceptionMessage)
{
  ArgumentUtility.CheckNotNull ("exceptionMessage", exceptionMessage);

  return new WebTestException ($"The element cannot be found: {exceptionMessage}");
}
```

Would this be desirable? There are **2** occurrences where a `MissingHtmlException` is re-thrown with this message, and **3** occurrences where an `AmbiguousException` is re-thrown with the message `$"Multiple elements were found: {exception.Message}"`.

Reply:
"exceptionMessage" -> "exceptionDetails"

I don't really understand the rest of the question with re-throw, etc. Something is missing for me to get it.